### PR TITLE
style: Adds "deprecated_" prefix to all non-theme colors

### PR DESCRIPTION
**Pull request by danielcolinjames**
This PR is a first step toward standardizing all colors in the codebase to adhere to the Uniswap Design System

It adds "`deprecated_`" before each color name so that it's clear in the code which colors need to be updated to one of the named colors from the design system

Here are all the colors being updated:

| old name | new name |
| -- | -- |
| white | deprecated_white |
| black | deprecated_black |
| text1 | deprecated_text1 | 
| text2 | deprecated_text2 | 
| text3 | deprecated_text3 | 
| text4 | deprecated_text4 | 
| text5 | deprecated_text5 | 
| bg0 | deprecated_bg0 | 
| bg1 | deprecated_bg1 | 
| bg2 | deprecated_bg2 | 
| bg3 | deprecated_bg3 | 
| bg4 | deprecated_bg4 | 
| bg5 | deprecated_bg5 | 
| bg6 | deprecated_bg6 | 
| modalBG | deprecated_modalBG | 
| advancedBG | deprecated_advancedBG |
| primary1 | deprecated_primary1 | 
| primary2 | deprecated_primary2 | 
| primary3 | deprecated_primary3 | 
| primary4 | deprecated_primary4 | 
| primary5 | deprecated_primary5 | 
| primaryText1 | deprecated_primaryText1 |
| secondary1 | deprecated_secondary1 | 
| secondary2 | deprecated_secondary2 | 
| secondary3 | deprecated_secondary3 | 
| red1 | deprecated_red1 | 
| red2 | deprecated_red2 | 
| red3 | deprecated_red3 |
| green1 | deprecated_green1 | 
| yellow1 | deprecated_yellow1 |
| yellow2 | deprecated_yellow2 |
| yellow3 | deprecated_yellow3 |
| blue1 | deprecated_blue1 | 
| blue2 | deprecated_blue2 | 
| error | deprecated_error | 
| success | deprecated_success | 
| warning | deprecated_warning |
| blue4 | deprecated_blue4 |

### CODE REVIEW

File: src/pages/Earn/index.tsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ export default function Earn() {
                 </ThemedText.White>
               </RowBetween>{' '}
               <ExternalLink
-                style={{ color: 'white', textDecoration: 'underline' }}
+                style={{ color: 'deprecated_white', textDecoration: 'underline' }}
```
Comment (vm) [inline src/pages/Earn/index.tsx:78]: accidental

File: src/pages/Pool/v2.tsx [hunk 2]
```diff
@@ -160,7 +160,7 @@ export default function Pool() {
                   </ThemedText.White>
                 </RowBetween>
                 <ExternalLink
-                  style={{ color: 'white', textDecoration: 'underline' }}
+                  style={{ color: 'deprecated_white', textDecoration: 'underline' }}
```
Comment (vm) [inline src/pages/Pool/v2.tsx:163]: revert

**Review by kayleegeorge — APPROVED** [review]

File: src/pages/Vote/Landing.tsx [hunk 3]
```diff
@@ -161,7 +163,10 @@ export default function Landing() {
                     </ThemedText.White>
                   </RowBetween>
                   <ExternalLink
-                    style={{ color: 'white', textDecoration: 'underline' }}
+                    style={{
+                      color: theme.deprecated_white,
```
Comment (vm) [inline src/pages/Vote/Landing.tsx:167]: why do we need a deprecated white?

File: src/pages/Pool/v2.tsx (hunk 2 above)
Comment (danielcolinjames) [reply src/pages/Pool/v2.tsx:163]: good catch, accidentally did this in one other place but missed these ones. should I update these to use `theme.deprecated_white` though?

Comment (vm) [reply src/pages/Pool/v2.tsx:163]: that works

File: src/pages/Vote/Landing.tsx (hunk 3 above)
Comment (vm) [reply src/pages/Vote/Landing.tsx:167]: gotcha, that works for me

**Review by vm — APPROVED** [review]

**Pull request merged**